api/json-vs-xml.md

```diff
@@ -1,31 +1,120 @@
+Both formats contain identical data structures and functionality - tXML is a one-to-one mapping of the JSON format.
 
-* [Send order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendOrderByBuyerRoute) and
-* [Send single delivery order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendSingleDeliveryOrderByBuyerRoute) endpoints support `tXML`.
+## When to use each format
 
-You can see an XML example by selecting "application/xml" in the "Parameter content type" dropdown, under the "Example Value" in above API specifications:
+### Use JSON when
+
+- Building new modern integrations (recommended)
+- Implementing REST API clients
+- You need smaller payload sizes
+
+### Use tXML when
+
+- Integrating with legacy systems that require XML
+- Working with enterprise middleware that requires XML
+- Your existing infrastructure is XML-based
+
+## JSON Format
+
+JSON is the **default and recommended format** for all Tradecloud API endpoints.
+
+### Advantages
+
+- Smaller payload size & faster parsing
+- Better human readability
+- Widely supported across programming languages
+
+### Implementation
+
+All API endpoints support JSON by default. Simply use `Content-Type: application/json` in your HTTP headers.
+
+For detailed JSON structure and examples, see the [JSON body format documentation](requests.md#json-body).
+
+## tXML Format
+
+tXML (Tradecloud XML) is a proprietary XML format that provides the same functionality as JSON but in XML structure.
+
+### Supported Endpoints
+
+The following API endpoints support tXML format:
+
+#### Buyer Endpoints
+
+- [Send order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendOrderByBuyerRoute)
+- [Send single delivery order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendSingleDeliveryOrderByBuyerRoute)
+
+#### Supplier Endpoints
+
+- [Send order response](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/supplier-endpoints/sendOrderResponseBySupplierRoute)
+
+#### Webhook Endpoints
+
+- [Order Webhook](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-webhook-connector/specs.yaml#/order-webhook%20endpoints/webhookPost)
+
+### Implementation
+
+To use tXML format, set the `Content-Type` header to `application/xml` in your HTTP requests.
+
+### Viewing tXML Examples
+
+You can view tXML examples in the Swagger UI documentation:
+
+1. Navigate to the desired endpoint in the API documentation
+2. In the "Parameter content type" dropdown, select "application/xml"
+3. The "Example Value" section will show the tXML structure
+
+#### API Endpoints
```

Comment:
This is duplicate of what is listed above.

Reply:
Removed, also removed the screenshots, as the Swagger UI will change